### DISCUSSION TITLE
Types of Ai

### DISCUSSION THREAD

**Discussion by ZaidHussain65** [ZaidHussain65/AIlab#3] [Q&A] [answered]
What are the different types of AI?

**Comment by Anashussain17** [comment, marked as answer]
The 7 types of artificial intelligence (AI) include:
Narrow AI or artificial narrow intelligence (ANI)
General AI or artificial general intelligence (AGI)
Super AI or artificial superintelligence (ASI)
Reactive machines.
Limited memory.
Theory of mind.
Self-aware.
